src/allotropy/parsers/beckman_pharmspec/beckman_pharmspec_reader.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, named_file_contents: NamedFileContents) -> None:
         self.header = SeriesData(header_data)
 
         data = df.loc[start:end, :]
-        data = data.dropna(how="all").dropna(how="all", axis=1)
+        data = data.dropna(how="all").dropna(how="all", axis="columns")
```

Comment:
Note to reviewer: this is unrelated cleanup. I had a PR a bit ago to do this rename everywhere, this must have slipped in since then.